calico_rkt/calico_rkt.py

```diff
@@ -429,6 +448,21 @@ def _find_ipam_plugin(self):
 
         return str(plugin_path)
 
+    def _set_error_response(self, code, message, details):
+        """Set the error_reponse class attribute with a json formatted reponse
+
+        :return: None
+        """
+        assert not self.error_response, "An error has already been hit " \
+                                        "during execution."
+        _log.error("Exiting with: %s", message)
+        error_response = json.dumps({
+            "cniVersion": "0.1.0", "code": code,
+            "message": message, "details": details
+        })
+        self.error_response = json.loads(error_response)
+        # Is there a reason to not call sys.exit here?
```

Comment:
The reason is so we can perform any cleanup we might need to do before exiting, and for a clearer method signature.